src/controllers/enter.information.controller.ts

```diff
@@ -32,5 +31,28 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     return next(e);
   }
 
-  return res.redirect(OBJECTIONS_DOCUMENT_UPLOAD);
+  const changingDetails = retrieveFromObjectionSession(session, CHANGE_ANSWER_KEY);
+  if (changingDetails) {
```

Comment:
I thought this was work was for another ticket OBJ-285.

Reply:
Honestly didn't notice that this was intended for a different sub-task, it's simple enough to add in though so may as well do both.